src/vs/workbench/contrib/terminal/browser/media/xterm.css

```diff
@@ -221,15 +221,18 @@
 	overflow-x: hidden;
 }
 
-.xterm .xterm-accessible-buffer div {
+.xterm-accessible-buffer div {
 	white-space: pre-wrap;
 }
 
-.xterm .xterm-accessible-buffer:focus-within {
+.xterm-accessible-buffer.active {
```

Comment:
should all of these be renamed to remove `xterm` now that it's not coming from `xterm`?